improver/cli/generate_timezone_mask_ancillary.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# (C) British Crown Copyright 2017-2020 Met Office.
+# All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#
+# * Redistributions of source code must retain the above copyright notice, this
+#   list of conditions and the following disclaimer.
+#
+# * Redistributions in binary form must reproduce the above copyright notice,
+#   this list of conditions and the following disclaimer in the documentation
+#   and/or other materials provided with the distribution.
+#
+# * Neither the name of the copyright holder nor the names of its
+#   contributors may be used to endorse or promote products derived from
+#   this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+# LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+# CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+# SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+# CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+"""Script to run GenerateTimezoneMask ancillary generation."""
+
+from improver import cli
+
+
+@cli.clizefy
+@cli.with_output
+def process(
+    cube: cli.inputcube, *, ignore_dst=True, time=None, groupings: cli.inputjson = None
+):
+    """Generate a timezone mask ancillary.
+
+    Create masks for regions of a geographic grid that are on different
+    timezones. The resulting masks can be used to isolate data for specific
+    timezones from a grid of the same shape as the cube used in this plugin
+    to generate the masks.
+
+    Args:
+        cube (iris.cube.Cube):
+            A cube with the desired grid. If no 'time' argument is provided
+            to the CLI, the time on this cube will be used for determining the
+            validity time of the calculated UTC offsets (this is only relevant
+            if daylights savings times are being included).
+        ignore_dst (bool):
+            If True, find and use the UTC offset to a grid point ignoring
+            daylights savings. Default is True.
```

Comment:
Possibly this should have been mentioned in #1301, I was thinking that it might be clearer to flip the logic here, so that this variable becomes something like `include_dst`. I think that would make the usage of this variable less prone to error.

Reply:
Done.